app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchUiRenderer.kt

```diff
@@ -11,5 +11,9 @@ class OverdueSearchUiRenderer(
       ui.showSearchHistory(model.overdueSearchHistory.orEmpty())
       ui.hideSearchResults()
     }
+
+    if (model.hasOverdueSearchResults) {
+      ui.showOverdueSearchResults(model.overdueSearchResults!!)
+    }
```

Comment:
This will run every time model changes, even if the search query is empty. So, we will still end showing the search results. Maybe we can handle it in the above conditional? WDYT?